### ISSUE TITLE
.NET Crash when trying to open last build

### ISSUE THREAD

**Issue by DualBandWiFi** [Xeroday/Spotify-Ad-Blocker#154]
Somehow it works on my work notebook, but not at home.

Here's the Event Log.

Small problem: it's on spanish 😆 

Aplicación: EZBlocker.exe
Versión de Framework: v4.0.30319
Descripción: el proceso terminó debido a una excepción no controlada. -> The process ended due to an unhandled exception
Información de la excepción: System.ComponentModel.Win32Exception
   en System.Diagnostics.ProcessManager.OpenProcess(Int32, Int32, Boolean)
   en System.Diagnostics.Process.GetProcessHandle(Int32, Boolean)
   en System.Diagnostics.Process.get_HasExited()
   en EZBlocker.SpotifyHook.IsRunning()
   en EZBlocker.SpotifyHook.<.ctor>b__21_0(System.Object)
   en System.Threading.ExecutionContext.RunInternal(System.Threading.ExecutionContext, System.Threading.ContextCallback, System.Object, Boolean)
   en System.Threading.ExecutionContext.Run(System.Threading.ExecutionContext, System.Threading.ContextCallback, System.Object, Boolean)
   en System.Threading.TimerQueueTimer.CallCallback()
   en System.Threading.TimerQueueTimer.Fire()
   en System.Threading.TimerQueue.FireNextTimers()

**Comment by Eddi2015** [comment]
I also get this error:

For me it's in German (btw I can help translating to German)
exception:

System.IO.DirectoryNotFoundException: Ein Teil des Pfades "C:\Users\Besitzer\AppData\Local\Temp\EZBlocker\2034" konnte nicht gefunden werden.
   bei System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   bei System.IO.Directory.Delete(String fullPath, String userPath, Boolean recursive, Boolean checkHost)
   bei EZBlocker.SpotifyPatcher.Patch()
   bei EZBlocker.Main.Main_Load(Object sender, EventArgs e)
   bei System.Windows.Forms.Form.OnLoad(EventArgs e)
   bei System.Windows.Forms.Form.OnCreateControl()
   bei System.Windows.Forms.Control.CreateControl(Boolean fIgnoreVisible)
   bei System.Windows.Forms.Control.CreateControl()
   bei System.Windows.Forms.Control.WmShowWindow(Message& m)
   bei System.Windows.Forms.Control.WndProc(Message& m)
   bei System.Windows.Forms.Form.WmShowWindow(Message& m)
   bei System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)

Maybe you can use a tool such as google translator or ask me if you don't understand sth German.

**Comment by DualBandWiFi** [comment]
That isn't the same error, mine is an unhandled exception and yours is because its trying to delete a directory which doesn't exists.
As a temp fix you could try to create that dir, it's where the localhost server script is located.

**Comment by Xeroday** [comment]
Have you both tried running as admin?

**Comment by DualBandWiFi** [comment]
Of course, I always run it as admin to avoid potential issues, at the end i trust on your software.

**Comment by DualBandWiFi** [comment]
Re-downloaded it, changed dir and suddenly it started working, let's see in some hours if it doesn't crashes.

**Comment by Eddi2015** [comment]
Well the proble is my anti Virus triggers this program. Even when I whitlist it.
